ibm/resource_ibm_cis_dns_records_import.go

```diff
@@ -0,0 +1,109 @@
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/IBM/go-sdk-core/v3/core"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	cisDNSRecordsImportFile               = "file"
+	cisDNSRecordsImportTotalRecordsParsed = "total_records_parsed"
+	cisDNSRecordsImportRecordsAdded       = "records_added"
+)
+
+func resourceIBMCISDNSRecordsImport() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			cisID: {
+				Type:        schema.TypeString,
+				Description: "CIS instance crn",
+				Required:    true,
+			},
+			cisDomainID: {
+				Type:             schema.TypeString,
+				Description:      "Associated CIS domain",
+				Required:         true,
+				DiffSuppressFunc: suppressDomainIDDiff,
+			},
+			cisDNSRecordsImportFile: {
+				Type:        schema.TypeString,
+				Description: "File to import",
+				Required:    true,
+				ForceNew:    true,
+			},
+			cisDNSRecordsImportTotalRecordsParsed: {
+				Type:        schema.TypeInt,
+				Description: "total records parsed",
+				Computed:    true,
+			},
+			cisDNSRecordsImportRecordsAdded: {
+				Type:        schema.TypeInt,
+				Description: "added records count",
+				Computed:    true,
+			},
+		},
+
+		Create:   resourceCISDNSRecordsImportUpdate,
+		Read:     resourceCISDNSRecordsImportRead,
+		Update:   resourceCISDNSRecordsImportRead,
+		Delete:   resourceCISDNSRecordsImportDelete,
+		Importer: &schema.ResourceImporter{},
+	}
+}
+func resourceCISDNSRecordsImportUpdate(d *schema.ResourceData, meta interface{}) error {
+	cisClient, err := meta.(ClientSession).CisDNSRecordBulkClientSession()
+	if err != nil {
+		return err
+	}
+
+	crn := d.Get(cisID).(string)
+	zoneID, _, err := convertTftoCisTwoVar(d.Get(cisDomainID).(string))
+	cisClient.Crn = core.StringPtr(crn)
+	cisClient.ZoneIdentifier = core.StringPtr(zoneID)
+	file := d.Get(cisDNSRecordsImportFile).(string)
+
+	f, err := os.Open(file)
```

Comment:
Instead of opening file inside resource.. 
Can it be file = `file(<filepath>)` in terraform